OS/DiskOS/terminal.lua

```diff
@@ -274,7 +283,10 @@ function term.loop() --Enter the while loop of the terminal
   checkCursor() term.prompt()
   buffer, inputPos = "", 1
   for event, a,b,c,d,e,f in pullEvent do
-    checkCursor() --Which also draws the cursor blink
+    --If an autocomplete suggestion is displayed, doesn't make the cursor blink
+    if next(autocompleteSuggestions) == nil then
```

Comment:
This expression could be simplified:
 Values in acts as booleans:
- false, nil -> false.
- true, number, table, string, userdata -> true.

So this expression could be written as:
```lua
if not next(autocompleteSuggestions) then
```